### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by Jessemccallums**
The section I built for this milestone is:
the contact form in the footer:

I made sure that:

+The name field accepts up to 30 chars.
+The email field accepts only emails.
+The message field accepts up to 500 chars.
+All fields have placeholder text.
+All fields have the correct type.
+A user cannot submit the form if one or more of the fields is empty.
+I collected data by using the [Formspree service](https://formspree.io/html).

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -353,6 +371,21 @@ button:hover {
   grid-template-rows: auto;
 }
 
+form {
+  margin: auto;
+  justify-content: center;
+  text-align: center;
+}
```
Comment (vechicin) [inline main.css]: According to the project requirements, the form layout should be rendered using flexbox.

Screen Shot 2022-05-31 at 8 33 42

Your form is using grid, kindly fix this.

File: index.html [hunk 2]
```diff
@@ -9,222 +9,249 @@
...
+                <li class="styling1"><img src="Vector.1.svg" alt="icon"></li>
+                <li class="styling1"><img src="Group.png" alt="icon"></li>
+                <li class="styling1"><img src="Vector.3.svg" alt="icon"></li>
+                <li class="styling1"><img src="Vector.icon.svg" alt="icon"></li>
+                <li class="styling1"><img src="Vector.5.svg" alt="icon"></li>
+    
+            </ul>
+        </div>
+        <p class="text-footer">jessemccallums2001@gmail.com</p>
+    </footer>
```
Comment (vechicin) [inline index.html]: Your design doesn't look quite like the template design.

Here's your design:

Screen Shot 2022-05-31 at 8 39 16

Here's the template design:

Screen Shot 2022-05-31 at 8 40 07

Do you see how different margins, padding, heights, and the overall layout are? Kindly fix that.

Additionally, there's a strange horizontal scrolling behavior you should fix as well.

File: index.html [hunk 3]
```diff
@@ -9,222 +9,249 @@
...
+    <h1 class="maintextfooter">Contact us</h1>
+    <p class="paragraph-text1">If you have an application you are interested in developing, a feature that you need built or a project that
+        needs coding. I’d love to help with it </p>
+    <footer class="third width">
+        <form action="https://formspree.io/f/mgeddrqw"
+        method="POST">
+            <input type="text" name="name" id="name" placeholder="YourName" class="input123" maxlength="30" required=""><br>
+            <input type="email" name="email" id="email" placeholder="youremail@gmail.com" class="input1234" required="" ><br>
+            <textarea name="Message" id="Message" cols="30" rows="10" placeholder="Leave your message" class="input12345" maxlength="500" required=""></textarea>
+            <button class="button12">See project</button>
```
Comment (vechicin) [inline index.html:240]: Although your button has three interaction stylings (default, hover, active), they don't look quite like the ones from the design. You need to implement the disabled styling as well. Kindly fix this.

Screen Shot 2022-05-31 at 8 44 12

Screen Shot 2022-05-31 at 8 44 51

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @Jessemccallums  ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Good PR title/description
- [x] Proper form implementation

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Jessemccallums) [reply index.html]: That's weird, here in my browser looks like this:

image

File: index.html (hunk 3 above)
Comment (Jessemccallums) [reply index.html:240]: Hi Sir, I already solved it, check the next pull request

File: main.css (hunk 1 above)
Comment (Jessemccallums) [reply main.css]: Hi Sir, I already solved it, check the next pull request

File: index.html (hunk 2 above)
Comment (Jessemccallums) [reply index.html]: Hi Sir, I already solved it, check the next pull request

File: index.html [hunk 4]
```diff
@@ -9,222 +9,251 @@
...
+            </ul>
+
+            <button type="button" class="button1">See project</button>
+
+
+        </div>
+    </section>
+    
+    <footer class="third width">
+    <h1 class="maintextfooter">Contact us</h1>
```
Comment (tufoinnkuo10) [inline index.html:231]: - [ optional ] you can make your contact us heading bolder so it looks like the figma version

File: index.html [hunk 5]
```diff
@@ -9,222 +9,251 @@
...
+    <p class="paragraph-text1">If you have an application you are interested in developing, a feature that you need built or a project that
+        needs coding. I’d love to help with it </p>
+    
+        <form action="https://formspree.io/f/mgeddrqw"
+        method="POST" class="form-center" >
+            <input type="text" name="name" id="name" placeholder="YourName" class="input123" maxlength="30" required=""><br>
+            <input type="email" name="email" id="email" placeholder="youremail@gmail.com" class="input1234" required="" ><br>
+            <textarea name="Message" id="Message" cols="30" rows="10" placeholder="Leave your message" class="input12345" maxlength="500" required=""></textarea>
+            <button class="button12">See project</button>
```
Comment (tufoinnkuo10) [inline index.html:240]: - [ optional]please add the submit type to your button

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Jessemccallums 

Great job implementing the changes as requested by the previous reviewer. nevertheless, there are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ contact form submits successfully

Required Changes ♻️
Check the comments under the review
- [ Required] please like the previous reviewer stated, there is a strange horizontal movement from left to right with your app, please fix that and make it centered or steady. 😄 
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by topeogunleye — APPROVED** [review]
### Hi @Jessemccallums, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
> 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**